src/pudl/package_data/eia923/column_maps/energy_storage.csv

```diff
@@ -0,0 +1,71 @@
+net_generation_mwh_april,netgen_april,netgen_april,netgen_april,netgen_april,netgen_april,netgen_april,netgen_april,netgen_april,netgen_april,netgen_april
+net_generation_mwh_may,netgen_may,netgen_may,netgen_may,netgen_may,netgen_may,netgen_may,netgen_may,netgen_may,netgen_may,netgen_may
+net_generation_mwh_june,netgen_june,netgen_june,netgen_june,netgen_june,netgen_june,netgen_june,netgen_june,netgen_june,netgen_june,netgen_june
+net_generation_mwh_july,netgen_july,netgen_july,netgen_july,netgen_july,netgen_july,netgen_july,netgen_july,netgen_july,netgen_july,netgen_july
+net_generation_mwh_august,netgen_august,netgen_august,netgen_august,netgen_august,netgen_august,netgen_august,netgen_august,netgen_august,netgen_august,netgen_august
+net_generation_mwh_september,netgen_september,netgen_september,netgen_september,netgen_september,netgen_september,netgen_september,netgen_september,netgen_september,netgen_september,netgen_september
+net_generation_mwh_october,netgen_october,netgen_october,netgen_october,netgen_october,netgen_october,netgen_october,netgen_october,netgen_october,netgen_october,netgen_october
+net_generation_mwh_november,netgen_november,netgen_november,netgen_november,netgen_november,netgen_november,netgen_november,netgen_november,netgen_november,netgen_november,netgen_november
+net_generation_mwh_december,netgen_december,netgen_december,netgen_december,netgen_december,netgen_december,netgen_december,netgen_december,netgen_december,netgen_december,netgen_december
+total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consumption_quantity
+electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity,electric_fuel_consumption_quantity
+gross_generation_mwh,gross_generation_megawatthours,gross_generation_megawatthours,gross_generation_megawatthours,gross_generation_megawatthours,gross_generation_megawatthours,gross_generation_megawatthours,gross_generation_megawatthours,gross_generation_megawatthours,gross_generation_megawatthours,gross_generation_megawatthours
+net_generation_mwh,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours
```

Comment:
I mention this in #3512, but the `generation_fuel` table contains this same column with `megawatthours` written out instead of abbreviated. I think it should be abbreviated in both.

If you agree, I'll go ahead and change it in the `generation_fuel` table.

Reply:
the `net_generation_megawatthours` is a weird holdover. the column is immediately dropped in `_core_eia_923__generation_fuel_eia923`. in both these tables, this column is the total for the whole year, where the `net_generation_mwh_{month}` is the monthly data. we convert the monthly columns into the mwh column via [_yearly_to_monthly_records](https://github.com/catalyst-cooperative/pudl/blob/b57dfad8cb7427815656e0bfb90d35bae012fc26/src/pudl/transform/eia923.py#L386).

I think if you name this `net_generation_mwh` I think it'll junk up the yearly to monthly process. Maybe we should name this `net_generation_mwh_total`? But regardless this guy is gonna be dropped.

Reply:
Ok I like the idea of `mwh_total`. Do we vet the sum of the months with the total at all or do we just drop the total in favor of the summed months?

Reply:
we do not use the total column to check the monthly data. it would be a good idea overall but we definitely drop the annual total. checking would be good but oos for this for sure.

Reply:
agreed! good to know though